packages/http/src/validator/validators/__tests__/__snapshots__/body.spec.ts.snap

```diff
@@ -1,13 +1,12 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`HttpBodyValidator validate() body schema is provided validator for given media type exists return validation errors 1`] = `
+exports[`HttpBodyValidator validate() body schema is provided return validation errors 1`] = `
 Array [
   Object {
-    "code": "testName",
-    "message": "testMessage",
+    "code": "type",
```

Comment:
This was caused because the mocked functions apparently were not restored correctly, thous returning a not correct error